app/lending/(1-concepts)/collateralisation/page.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Collateral & Liquidations
+description: How LTV, collateralisation, and liquidations work in Chainflip Lending
+---
+
+## Collateralisation & Liquidations (LTV & Liquidation Thresholds)
+
+### Collateralisation and LTV
+
+Chainflip Lending uses external oracle prices to determine loan collateralisation and liquidation thresholds. This prevents internal pool price movements from affecting loan safety. Because Chainflip supports only highly liquid assets, concerns often associated with oracle dependency are significantly reduced.
+
+Collateral values are sourced from Chainlink feeds witnessed by the validator network across Ethereum and Arbitrum for redundancy. These oracle prices determine loan health through the Loan-to-Value ratio (LTV):
+
+<div style={{ textAlign: "left" }}>
+$$
+\text{LTV} = \frac{\text{Loan Value}}{\text{Collateral Value}}
+$$
+</div>
+
+A minimum collateralisation level is required (enforcing maximum LTV) when creating a loan. Borrowers can freely add more collateral at any time. If a user maintains an excessively low LTV (high overcollateralisation), a gradually increasing Low LTV Penalty is applied. This discourages locking up unnecessary collateral that reduces system-wide economic security.
```

Comment:
Is it worth adding that the penalty is up to 1% APR to show that it is relatively low? Might even be worth giving them the whole "formula": this penalty is 0% at 50% LTV and linearly increases up to 1% as LTV approaches 0%.